net/HttpRequest.hpp

```diff
@@ -441,6 +464,30 @@ class Header
     /// Return true iff Transfer-Encoding is set to chunked (the last entry).
     bool getChunkedTransferEncoding() const { return _chunked; }
 
+    bool hasConnectionToken() const { return has(CONNECTION); }
+    ConnectionToken getConnectionToken() const {
+        const std::string token = get(CONNECTION);
+        if( Util::iequal("close", token) ) {
+            return ConnectionToken::Close;
+        } else if( Util::iequal("keep-alive", token) ) {
+            return ConnectionToken::KeepAlive;
+        } else if( Util::iequal("upgrade", token) ) {
+            return ConnectionToken::Upgrade;
+        } else {
+            return ConnectionToken::None;
+        }
+    }
+    void setConnectionToken(ConnectionToken token) {
+        std::string value;
+        switch( token ) {
+            case ConnectionToken::Close: value = "close"; break;
```

Comment:
Nit: More readable to break these on multiple lines. The reader's time and attention are more valuable resources than a line or two on the screen (let alone in the file). (Don't force me to parse 3 statements on one line, please.)